src/core/team-core.js

```diff
@@ -4,22 +4,55 @@ import {deepChangeKeyCase} from '../util';
 
 function getTeams(opts) {
   const isBanned = opts.client && !!opts.client.isBanned;
+  const voteScoreSql = `
+    (SELECT
+      team_id,
+      SUM(value) AS value
+    FROM
+      (SELECT
+        wilsons(
+            COUNT(CASE votes.value WHEN 1 THEN 1 ELSE null END)::int,
+            COUNT(CASE votes.value WHEN -1 THEN -1 ELSE null END)::int
+          ) * COUNT(votes)::numeric AS value,
+        users.team_id AS team_id
+      FROM feed_items
+      LEFT JOIN users ON users.id = feed_items.user_id
```

Comment:
I think this can be changed to normal `JOIN`. Only system messages don't have a user and we don't want to include those anyway